README.md

```diff
@@ -49,7 +49,8 @@ For an M1 Mac, read this [SO](https://stackoverflow.com/c/expensify/questions/11
 
 ## Running the Android app 🤖
 * To install the Android dependencies, run: `npm install`
-* Go through the instructions on [this page](https://reactnative.dev/docs/environment-setup#development-os) for "React Native CLI Quickstart" > Mac OS > Android
+* Go through the instructions on [this SO post](https://stackoverflow.com/c/expensify/questions/13283/13284#13284) to start running the app on android.
```

Comment:
Not sure if the external contributors will have access to the SO link or not. (I guess they do not have access) so do you want to mention that?

Reply:
Oh ok, I didn't know that they don't have access. I think pasting the instructions in the readme might be too long but is there anyway to make the SO public or is there another way to allow them to see the instructions?

Reply:
I guess we will have to add it to README, I am not sure if we have an alternative.

Reply:
Any update on this one?

Reply:
Hey Vivek, sorry was OOO this past week. I'm thinking that we can merge the pull request as is since the same thing is currently in the "Running the iOS App" section of the Readme : https://github.com/Expensify/App/blob/f856cce68b97f8fe0a3cf0b4d51b0567459eac72/README.md It also links an SO for cocoapods. External contributors will still be able to run android without this SO but I think it just makes it easier for employees.

Reply:
Alright, let's merge it then. Thanks